expr/ztests/cut-to-root-error-unset-value.yaml

```diff
@@ -3,6 +3,4 @@ zed: 'cut this:=a'
 input: |
   {a:null({b:int32})}
 
-warnings: |
-  cut: cannot cut an unset value to .
-  cut: no record found with columns a
+errorRE: "cut: 'this' not allowed \\(use record literal\\)"
```

Comment:
I think we should just keep one of cut-to-root-error-non-record.yaml and cut-to-root-error-unset-value.yaml, and we should rename it to cut-to-root-error.yaml.